pygmo/docstrings.cpp

```diff
@@ -1602,7 +1602,7 @@ The adaptation of the mutation variant is added to :class:`~pygmo.core.sade`
 
 Args:
     gen (``int``): number of generations
-    allowed_variants (``NumPy array or list of floats``): allowed mutation variants, each one being a number in [1, 18]
+    allowed_variants (``NumPy array or list of int``): allowed mutation variants, each one being a number in [1, 18]
```

Comment:
I'd put only ``list`` and ``int`` in the backticks, as the convention elsewhere is that the backticks are used for Python entities (``float``, ``int``, ``True``, ``None``, etc.). Also, what kind of objects are exactly allowed in input here? Would tuples work? Other types of iterable objects?